octopuslite/utils.py

```diff
@@ -179,18 +182,18 @@ def image_generator(files, crop: Optional[tuple] = None):
         E.g. image_generator(DaskOctopusLiteLoader.files('channel name'))
     crop : tuple, optional
         An optional tuple which can be used to perform a centred crop on the
-        data.
+        image data.
     """
     shape = imread(files[0]).shape
     dims = imread(files[0]).ndim
-    if crop == None:
+    if crop is None:
         for filename in files:
             img = imread(filename)
             yield img
     else:
         cslice = lambda d: slice(
-            int((shape[d] - crop[d]) // 2),
-            int((shape[d] - crop[d]) // 2 + crop[d]))
+            int((shape[d] - crop[d]) // 2), int((shape[d] - crop[d]) // 2 + crop[d])
+        )
         crops = tuple([cslice(d) for d in range(dims)])
```

Comment:
Since we're using this crop code in a few places, can we pull this out as a separate function in utils, e.g.

```python
def crop_image():
    ...
```

That way all occurances use the same machinery.

Reply:
would you like it to actually crop the image or just provide the cropping slice?

Reply:
(i did cropped image)